openconf.cfg

```diff
@@ -31,4 +31,4 @@ source libs/openconf.cfg
 source drivers/openconf.cfg
 source emulators/openconf.cfg
 source tools/openconf.cfg
-source ../poc/tcmod/openconf.cfg
+source #TCDIR/tcmod/openconf.cfg
```

Comment:
not $(TCDIR)?

Reply:
nope that does not work. sed is not able to change the $TCDIR name, rather it tries to substitute it.